active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+        * If the buttons have text properties set by the developer, narrator will announce that text instead of the default announcement. 
+* When the focus is on a number button, narrator will announce "page x of y". 
+* Combo box and number box modes will use the default narrator announcements for combo box and number box when the user puts focus on them. 
+* When the user enters a value in to the number box, narrator will announce the number entered. 
+
+### GamePad 
+* Pager Control will not have engagement. The internal components (ComboBox and NumberBox) will keep their current engagement and accessibility features for game pad. 
+* The user can use the right and left arrow keys to navigate between the buttons and use the A button to select an item. 
+* Guidance will be added for app developers that are targeting Xbox applications to not place items to the left or right of the pager control. 
+        * Text added to guidance: “If you are developing an application that will target Xbox, we recommend not placing any UI elements to the left or right of the pager control so the end user does not have to navigate through the pager control in order to reach the next UI element.” 
+
+### Localization
+* The numbers will follow standard behavior when localized to/from a language that uses a numeral system that differs from Western Arabic. 
+* The prefix and suffix text will follow standard mirroring behavior when localized to/from languages that are RTL or LTR. 
+
+## Data and Intellegence Metrics 
+
+Adoption and validation of the control in the community
+* Measurement: Use telemetry to determine how many first and third party developers are using the control in preview. 
+* Measurement: Count of posts providing feedback to us through Github and Discord. 
+
+Questions that can be answered through telemetry metrics to help focus future features and improvements: 
+Which display mode is the most commonly used? 
+* Measurement: Count of how many applications include the pager control type broken out by display mode.
+
+How often are developers using this control with an indefinite number of pages?
+* Measurement: Count of how many applications include the pager control type and have the NumberOfPages property empty.
+
+
+## Open Questions 
+* Should the combo box mode be an editable combo box? My worry with making it an editable combo box is the validity of having the number box mode. Would it hurt to have both editable combo box and number box as options?
+* I use the terms index and page throughout this document and the APIs. The APIs are all index while the documentation is mostly using page. Should I update the APIs to use page instead of index?
+* There are certain APIs that are specific to one of the three display modes and they will be ignored if the display mode chosen does not support those APIs. Is that something that needs to be reconsidered so it does not cause confusion for the developers?
```

Comment:
The only way I could think of, on how we would avoid this, is to have a configuration object that get's passed to the PagerControl. There would be three configuration object types, one for reach display mode and each configuration object only holds fields that are relevant for its display mode.

While this would avoid having properties be irrelevant based on the display mode, this would make it definitely harder to use that control since changing a simple property might get more complex than necessary. @ranjeshj @MikeHillberg What are your thoughts on the concept of a configuration object?

Reply:
Agree that a configuration object would be the most likely solution, and agree that it sounds more complicated than the problem it solves. We should call out each of these in the API Notes, maybe put a "note:" in front of each one to make it possible to look at that table and visualize the extent of the issue.

Reply:
Such a configuration object surely will be new for developers and a bit complicated. Would it be good to try to have some form of "experiment control" where we explore configuration objects further? It's a bit difficult to see how hard/bad the actual API with configuration objects would look like without actually having tried such a pattern out.